src/scene/vcViewpoint.cpp

```diff
@@ -121,8 +121,8 @@ void vcViewpoint::Cleanup(vcState * /*pProgramState*/)
 
 void vcViewpoint::SetCameraPosition(vcState *pProgramState)
 {
-  pProgramState->camera.position = m_CameraPosition;
-  pProgramState->camera.headingPitch = m_CameraHeadingPitch;
+  pProgramState->pActiveViewport->camera.position = m_CameraPosition;
+  pProgramState->pActiveViewport->camera.headingPitch = m_CameraHeadingPitch;
```

Comment:
The base function from `vcSceneItem` sets all of the viewports, not just the active one.